frontend/src/views/Lobby.tsx

```diff
@@ -639,6 +660,54 @@ function LobbyPage() {
                 />
               </SliderContainer>
             </HoverContainerSlider>
+            <NoMarginMediumText>Room Size</NoMarginMediumText>
+            <NoMarginSubtitleText>
+              {size === 31? 'No limit' : (size === 1? '1 person' : `${size} people`)}
```

Comment:
really minor nit:
```suggestion
              {size === 31 ? 'No limit' : (size === 1 ? '1 person' : `${size} people`)}
```

Reply:
Additionally, this gives me an `ESLint: Do not nest ternary expressions.(no-nested-ternary)` warning, so we might need a different way to do this to avoid the nested ternary (maybe putting the second ternary inside the template string?)